CHANGELOG

```diff
@@ -1,6 +1,125 @@
 devel
 -----
 
+* ArangoDB now has an optional AQL query execution plan cache that can be used
+  to skip query planning and optimization when running the same queries 
+  repeatedly. This feature is currently experimental.
+
+  The plan cache is fully optional and is only considered for AQL queries that
+  have the option `optimizePlanForCaching` set to `true`. If the option is not
```

Comment:
I thought we changed that to `enablePlanCache`.

Reply:
Forgot to change the CHANGELOG, will adjust.